SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialOSDispatcherInterface.h

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+
+#include "EngineClasses/SpatialActorChannel.h"
+#include "Schema/RPCPayload.h"
+
+#include <WorkerSDK/improbable/c_schema.h>
+#include <WorkerSDK/improbable/c_worker.h>
+
+DECLARE_DELEGATE_OneParam(EntityQueryDelegate, const Worker_EntityQueryResponseOp&);
+DECLARE_DELEGATE_OneParam(ReserveEntityIDsDelegate, const Worker_ReserveEntityIdsResponseOp&);
+DECLARE_DELEGATE_OneParam(CreateEntityDelegate, const Worker_CreateEntityResponseOp&);
+
+DECLARE_MULTICAST_DELEGATE_OneParam(FOnEntityAddedDelegate, const Worker_EntityId);
+DECLARE_MULTICAST_DELEGATE_OneParam(FOnEntityRemovedDelegate, const Worker_EntityId);
+
+class SpatialOSDispatcherInterface
```

Comment:
Would it be confusing to have `SpatialOSDispatcherInterface` and `SpatialDispatcher` in UnrealGDK?

Reply:
Yes, I agree it's confusing, especially considering the actual SpatialDispatcher doesn't inherit from the interface. The difference between the two is that the SpatialOSDispatcherInterface is dispatching network traffic, while the SpatialDispatcher is actually focused on callback dispatching.  If you have a different suggestion I'd be happy to change it, but I couldn't really come up with anything better. I'm hoping that as part of integrating worker wrapper we can clean up our naming.